tests/templates/pr-create.md.ejs.js

```diff
@@ -7,6 +7,8 @@
 
 "use strict";
 
+/* global describe, it */
```

Comment:
I'm not sure why this file needs this globals declaration while others don't (e.g. those in `test/lib/formatters`).